pytorch/inference/docker/1.5.1/py3/Dockerfile.neuron

```diff
@@ -1,4 +1,154 @@
 FROM ubuntu:16.04
 
-# Placeholder Dockerfile for Neuron
+LABEL dlc_major_version="1"
+LABEL maintainer="Amazon AI"
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
```

Comment:
The TF 1.15 dockerfile for Neuron support multi-model inference: https://github.com/aws/deep-learning-containers/blob/master/tensorflow/inference/docker/1.15/py3/Dockerfile.neuron

Please verify if this label is applicable here.

Reply:
Ok. This is not needed and hence removed it.